pkg/user/apiserver/registry/user/etcd/etcd.go

```diff
@@ -3,8 +3,7 @@ package etcd
 import (
 	"context"
 	"errors"
-	"strings"
-
+	"github.com/openshift/oauth-apiserver/pkg/oauth/apis/oauth/validation"
```

Comment:
move among the other openshift imports.

My preferred import order is:
system
k8s
openshift
local

Reply:
Do we have a code formatting settings somewhere? Sorting import manually is probably the last thing I should do...

Reply:
My IDE is capable of doing that on its own once the main groups have been manually created. This is the same as you'd do with Python library imports, too

Reply:
I manually sorted this out. Although, without a proper formatter this problems will be getting back over and over.